demisto_sdk/main.py

```diff
@@ -272,6 +273,23 @@ def format(file_type, **kwargs):
     return 1
 
 
+@main.command(name="init", short_help="Initiate a new Pack, Integration or Script.")
```

Comment:
Initiate a new Pack, Integration or Script. If the script/integration flags are not present then we will create a pack with the given name. Otherwise when using the flags we will generate a script/integration based on your selection.

Reply:
done